### DISCUSSION TITLE
Notes are randomly locked

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1571] [Q&A] [answered]
There's something I'm not quite understanding. Sometimes, I create a new note and a moment after it's locked and I have to click on Edit in order to be able to edit the note again.
Is there something I need to be aware of in order to avoid this from happening?

Thanks

**Comment by zadam** [comment, marked as answer]
Hi, see https://github.com/zadam/trilium/wiki/Text-notes#read-only-vs-editing-mode

Your note is probably big and therefore Trilium chooses to first present it in read only mode as an optimization.

**Reply by hokutonanto** [reply]
You're right, that's what happened.
